src/web/services/search.service.ts

```diff
@@ -359,19 +362,21 @@ export class SearchService {
     const distinctCourseIds: string[] = Object.keys(distinctInstructorsMap);
     const instructorsArray: Instructors[] = Object.values(distinctInstructorsMap);
     return forkJoin(
-      of(distinctCourseIds),
-      forkJoin(instructorsArray.map((instructors: Instructors) => {
-        return forkJoin(
-          instructors.instructors.map(
-            (instructor: Instructor) => this.instructorService.loadInstructorPrivilege(
-              {
-                courseId: instructor.courseId,
-                instructorId: instructor.googleId,
-              },
-            ),
-          ),
-        );
-      })),
+      instructorsArray.length === 0
+        ? of([])
+        : of(distinctCourseIds),
+          forkJoin(instructorsArray.map((instructors: Instructors) => {
```

Comment:
The alignment is not correct?

Reply:
👀 sorry mistakenly committed this code, this should not be that, removed it thanks